test/py/test_request_errors.py

```diff
@@ -189,24 +185,29 @@ def test_disconnected_socket_quiesce_busy(mock_quiesce):
 
 @patch('libvfio_user.reset_cb')
 @patch('libvfio_user.quiesce_cb', side_effect=fail_with_errno(errno.EBUSY))
-@patch('libvfio_user.migr_get_pending_bytes_cb')
-def test_reply_fail_quiesce_busy(mock_get_pending_bytes, mock_quiesce,
+@patch('libvfio_user.migr_trans_cb')
+def test_reply_fail_quiesce_busy(mock_migr_trans_cb, mock_quiesce,
                                  mock_reset):
     """Tests failing to reply and the quiesce callback returning EBUSY."""
 
     global ctx, sock
 
-    def get_pending_bytes_side_effect(ctx):
+    def migr_trans_cb_side_effect(ctx, state):
         sock.close()
         return 0
-    mock_get_pending_bytes.side_effect = get_pending_bytes_side_effect
-
-    # read the get_pending_bytes register, it should close the socket causing
-    # the reply to fail
-    read_region(ctx, sock, VFU_PCI_DEV_MIGR_REGION_IDX,
-                vfio_user_migration_info.pending_bytes.offset,
-                vfio_user_migration_info.pending_bytes.size, rsp=False,
-                busy=True)
+    mock_migr_trans_cb.side_effect = migr_trans_cb_side_effect
+
+    # change the state, it should close the socket causing the reply to fail
+    feature = vfio_user_device_feature(
```

Comment:
Maybe I've wrong but after having reviewed all this code I start feeling this is getting repeated, if that's the case then could we create helper function?